src/scenes/Event/components/EventTxHistory/index.js

```diff
@@ -12,13 +11,8 @@ import {
   withStyles,
 } from '@material-ui/core';
 
+import TxRow from './txRow';
```

Comment:
please name `./txRow` as PascalCase like `./TxRow`. To stay consistent with most of our app, let's make it this way please :)